### PR TITLE
Fixes #6804 - add note about misconfigured policy

### LINKED ISSUE #6804: Additional error text for troubleshooting step - Group policy misconfiguration
In the section "HOW TO ENABLE LISTENERS BY USING A GROUP POLICY", it explains that group policy may be the cause of the failure.

I suggest adding this error text as a possible symptom of this cause from a client machine where group policy is misconfigured:

`The client cannot connect to the destination specified in the request. Verify that the service on the destination is running and is accepting requests.`

And, that misconfiguration of the group policy (having a blank value for the filter)  can be quickly identified with the following command on the remote machine:

`winrm e winrm/config/listener` 

And looking for **ListeningOn=null**

References
- https://stackoverflow.com/questions/17281224/configure-and-listen-successfully-using-winrm-in-powershell
- http://pgeorgiev.com/winrm-would-not-listen-on-port-5985/

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 22d1b75d-4145-cca7-9ca9-89c724a0211f
* Version Independent ID: a76fca56-4886-3479-600b-4f3775e42c5c
* Content: [about_Remote_Troubleshooting - PowerShell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_remote_troubleshooting?view=powershell-7)
* Content Source: [reference/7.0/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.0/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1787994](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1787994) - Fixes #6804 - add note about misconfigured policy

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md [hunk 1]
```diff
@@ -139,38 +137,39 @@ WS-Management service is running on the remote host and configured to listen
...
-automatic configuration of listeners" policy in the following Group Policy
+To configure the listeners for all computers in a domain, enable the **Allow
+automatic configuration of listeners** policy in the following Group Policy
 path:
 
-Computer Configuration\\Administrative Templates\\Windows Components
-\\Windows Remote Management (WinRM)\\WinRM service
+```
+Computer Configuration\Administrative Templates\Windows Components
+\Windows Remote Management (WinRM)\WinRM service
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md:146]: I think this is a single path, but it got broken into two lines.  Maybe recombine?

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md [hunk 2]
```diff
@@ -179,59 +178,88 @@ Set-NetFirewallRule -Name "WINRM-HTTP-In-TCP-PUBLIC" -RemoteAddress Any
...
 To enable a firewall exception for in all computers in a domain, enable the
-"Windows Firewall: Allow local port exceptions" policy in the following
-Group Policy path:
+**Windows Firewall: Allow local port exceptions** policy in the following Group
+Policy path:
 
-Computer Configuration\\Administrative Templates\\Network
-\\Network Connections\\Windows Firewall\\Domain Profile
+```
+Computer Configuration\Administrative Templates\Network
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md:204]: Same here... this is a single path... should it be one line or the second line indented to indicate such?

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md [hunk 3]
```diff
@@ -662,44 +674,43 @@ contains unsigned script files and formatting files.
...
+execution policy. The execution policy change affects only the current process
+and does not change the PowerShell **ExecutionPolicy** registry setting.
 
 ```powershell
 Set-ExecutionPolicy -Scope process -ExecutionPolicy RemoteSigned
 ```
 
-You can also use the ExecutionPolicy parameter of **PowerShell.exe** to start
-a single session with a less restrictive execution policy.
+You can also use the **ExecutionPolicy** parameter of **PowerShell.exe** to start a
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md:694]: Shouldn't PowerShell.exe be in backticks?

**Review by DCtheGeek — CHANGES_REQUESTED** [review]

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md:146]: Stet - broken due to About_ topic line limitations

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md:204]: stet

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md (hunk 3 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md:694]: fixed

**Pull request merged**